scripts/run_py_sc_server.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+source set_vars.sh
```

Comment:
This is current-working-directory dependent, so it's best to use `${BASH_SOURCE[0]}` (the way it's happening in `install_py_sc_server.sh` to avoid confusing behavior.